client/utils/utils.go

```diff
@@ -1,10 +1,22 @@
 package utils
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/client/keys"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authctx "github.com/cosmos/cosmos-sdk/x/auth/client/context"
+	amino "github.com/tendermint/go-amino"
+)
+
+// GasAdjustment is applied to gas estimates to avoid tx
+// execution failures due to state changes that might
+// occur between the tx simulation and the actual run.
+const (
+	SimulateGasLimit      = 200000
+	GasEstimateAdjustment = 1.2
```

Comment:
Can this be a CLI flag (1.2 is probably a fine default value)?

Reply:
How about `--gas-adjustment`?